libopenthread/Makefile

```diff
@@ -0,0 +1,23 @@
+.PHONY: clean default
+default:
```

Comment:
I haven't tested using this makefile, but it should just build everything if you type `make`.

Reply:
Only reason this is empty was so that you explicitly specify `make mtd` or `make ftd`. Perhaps printing an error message prompting to run `make mtd` or `make ftd` would suffice?

Reply:
Why not just build both?